Tasks/AzureRmWebAppDeployment/AzureRMUtil.ts

```diff
@@ -0,0 +1,293 @@
+/// <reference path="../../definitions/node.d.ts" />
+/// <reference path="../../definitions/q.d.ts" />
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+
+var adal = require ('adal-node');
+
+import taskLib = require ('vsts-task-lib/task');
+var request = require ('request');
+import Q = require ('q');
+var parseString = require('xml2js').parseString;
```

Comment:
when should we do 'var' and when should we do 'import'?
if there is no difference - be consistent.
if there is a difference - group all vars and group all imports